src/main/java/io/reactivex/internal/operators/flowable/FlowableFromIterable.java

```diff
@@ -326,7 +399,17 @@ void slowPath(long r) {
                         return;
                     }
 
-                    if (!it.hasNext()) {
+                    boolean hasNext;
```

Comment:
I'm joking of course, if you don't mind please rename previous `b` to `hasNext` (nit, but readability++)